packages/@n8n/db/src/repositories/role.repository.ts

```diff
@@ -0,0 +1,11 @@
+import { Service } from '@n8n/di';
+import { DataSource, Repository } from '@n8n/typeorm';
+
+import { Role } from '../entities';
+
+@Service()
+export class RoleRepository extends Repository<Role> {
```

Comment:
It will always be time to move later, but I'm just thinking if we should not introduce a "custom role" module or something 🤔